test/run_tests.py

```diff
@@ -67,6 +67,10 @@ def process_output(args):
         call(['python', os.path.join(env.RESOURCES_DIR, 'statuschecker.py'),
              os.path.join(env.RESULTS_DIR, 'output.xml')])
     rebot = 'rebot' if os.sep == '/' else 'rebot.bat'
+    try:
+        args.remove('--dryrun')
+    except TypeError:
+        pass
```

Comment:
First of all, `list.remove` raises `ValueError`, not `TypeError`, if the removed item doesn't exist. Secondly, it would be shorted and cleaner to use:

```python
if '--dryrun' in args:
    args.remote('--dryrun')
```